index.html

```diff
@@ -10,11 +10,13 @@
     <title>Get Set FOSS!</title>
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <!-- Favicon -->
+    <link rel="icon" href="assets/img/get-set-foss.ico" type="image/svg+xml"
+    />
     <!-- Icon -->
     <link
       href="https://cdn.lineicons.com/1.0.1/LineIcons.min.css"
-      rel="stylesheet"
-    />
+      rel="stylesheet"/>
```

Comment:
```suggestion
      rel="stylesheet"
    />
```

Reply:
@dojutsu-user requested changes made in index.html